packages/create-sitecore-jss/src/templates/nextjs-personalize/src/lib/page-props-factory/plugins/personalize.ts

```diff
@@ -0,0 +1,32 @@
+﻿import { GetServerSidePropsContext, GetStaticPropsContext } from 'next';
+import { Plugin } from '..';
+import { personalizeLayout } from './../../layout-personalizer';
```

Comment:
Minor, but should be able to shorten to 
```suggestion
import { personalizeLayout } from 'lib/layout-personalizer';
```